core/docz/src/components/Props.tsx

```diff
@@ -118,20 +121,28 @@ export const Props: SFC<PropsProps> = ({
   const componentName =
     filemetaName || get('displayName', component) || get('name', component)
 
-  const componentMatcher = (componentName: string, item: any) => {
-    const matchingPatterns = [
-      filename,
-      `/${componentName}.`,
-      `/${kebabCase(componentName)}.`,
-      `/${pascalCase(componentName)}.`,
-    ]
-    return !!matchingPatterns.find(pattern => item.key.includes(pattern))
+  if (!stateProps || stateProps.length === 0) {
+    console.warn(`Looks like no components have been parsed via docz.`)
+    return null
   }
 
-  const found =
-    stateProps &&
-    stateProps.length > 0 &&
-    stateProps.find(item => componentMatcher(componentName, item))
+  const keys: string[] = stateProps.map(item => item.key)
+
+  const matchingPatterns = [
+    filename,
+    `/${componentName}.`,
+    `/${kebabCase(componentName)}.`,
+    `/${pascalCase(componentName)}.`,
+  ]
+
+  debug(`Matching component props via ${matchingPatterns.join(', ')}:`)
+  debug(keys)
+
+  const found = keys.find(key => {
+    return !!matchingPatterns.find(pattern => key.includes(pattern))
```

Comment:
CircleCI is probably breaking here. I would recommend checking the PR again and analyzing where exactly it is breaking.